client/src/deploy/make.rs

```diff
@@ -0,0 +1,29 @@
+use clap::{App, ArgMatches, SubCommand};
+
+use super::creation_common;
+use crate::command::ClientCommand;
+
+pub struct MakeDeploy;
+
+impl<'a, 'b> ClientCommand<'a, 'b> for MakeDeploy {
+    const NAME: &'static str = "make-deploy";
+    const ABOUT: &'static str = "Constructs a deploy and outputs it to a file \
+    or stdout. As a file, the deploy can subsequently be signed by other \
+    parties and sent to a node, or signed with the sign-deploy subcommand";
+
+    fn build(display_order: usize) -> App<'a, 'b> {
+        let subcommand = SubCommand::with_name(Self::NAME)
+            .about(Self::ABOUT)
+            .display_order(display_order);
+        let subcommand = creation_common::apply_common_session_options(subcommand)
+            .arg(creation_common::output::arg(display_order));
```

Comment:
We're still passing an incorrect value for the display order here - that's the order for the subcommand, not its args.

I'd suggest adding an `Input` and `Output` variant to `creation_common::DisplayOrder`:
```rust
pub(super) enum DisplayOrder {
    ShowArgExamples,
    NodeAddress,
    Input,
    Output,
    SecretKey,
    ...
```
then we can just use those rather than passing in the display order via `arg()`